tests/common/snappi_tests/common_helpers.py

```diff
@@ -811,10 +811,19 @@ def enable_packet_aging(duthost, asic_value=None):
         duthost.command("docker exec syncd python /packets_aging.py enable")
         duthost.command("docker exec syncd rm -rf /packets_aging.py")
     elif "platform_asic" in duthost.facts and duthost.facts["platform_asic"] == "broadcom-dnx":
-        try:
-            duthost.shell('bcmcmd -n {} "BCMSAI credit-watchdog enable"'.format(asic_value))
-        except Exception:
-            duthost.shell('bcmcmd -n {} "BCMSAI credit-watchdog enable"'.format(asic_value[-1]))
+        # if asic_value is present, enable packet aging for specific asic_value.
```

Comment:
We may need similar changes for disable_packet_aging as well ?

Reply:
@amitpawar12  can you address this comment? seems we need same change there.

Reply:
@sdszhang - I agree that we might require this change in disable_packet_aging as well. But this change will be ONLY applicable for pre SAI 11.x versions. Anything with SAI 11 and above, will need different method to enable/disable credit-watchdog.  We can discuss this wider audience.

Reply:
@sdszhang , @vmittal-msft  - Added similar code for disabling credit watch dog.

Reply:
@amitpawar12 pls track the SAI 11 questions/changes in another issue.